src/dune_rules/coq/coq_config.ml

```diff
@@ -105,7 +105,7 @@ module Version = struct
   let impl_version bin =
     let* _ = Build_system.build_file bin in
     Memo.of_reproducible_fiber
-    @@ Process.run_capture_line ~display:!Clflags.display Process.Strict bin
+    @@ Process.run_capture_line ~display:Quiet Process.Strict bin
```

Comment:
Here we query the version of Coq. This is internal and does not need to be reported to the user.